audformat/core/database.py

```diff
@@ -487,18 +510,25 @@ def pick_tables(
             self,
             table_ids: typing.Union[str, typing.Sequence[str]],
     ):
-        r"""Pick tables by ID.
+        r"""Pick (miscellaneous) tables by ID.
 
         Args:
             table_ids: table IDs to pick
 
+        Raises:
+            audformat.errors.BadIdError: if a table with provided ID
+                does not exist in the database
+            RuntimeError: if a misc table
+                that is used as scheme(s)
+                is part of the non-picked tables
```

Comment:
Sounds a bit strange. Maybe: "if a misc table that is used in a scheme would be removed?"

Reply:
Changed accordingly.